src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1125,6 +1125,63 @@ protected static void VerifyOperationTreeForTest<TSyntaxNode>(CSharpCompilation
             additionalOperationTreeVerifier?.Invoke(actualOperation, compilation, syntaxNode);
         }
 
+        protected static void VerifyFlowGraphForTest<TSyntaxNode>(CSharpCompilation compilation, string expectedFlowGraph)
```

Comment:
I wonder if the logic for printing graph to string would make sense as a `Dump()` method in the product code. Seems useful for debugging.
That could be done later if not yet sure.